$MirOS: ports/devel/automake/1.9/patches/patch-doc_automake_texi,v 1.3 2005/10/02 11:46:28 tg Exp $
--- doc/automake.texi.orig	Sat Jul  9 09:28:22 2005
+++ doc/automake.texi	Sun Jun 25 00:19:19 2006
@@ -34,12 +34,12 @@ published by the Free Software Foundatio
 
 @dircategory Software development
 @direntry
-* automake: (automake).         Making Makefile.in's.
+* automake-1.9: (automake-1.9).         Making Makefile.in files.
 @end direntry
 
 @dircategory Individual utilities
 @direntry
-* aclocal: (automake)Invoking aclocal.          Generating aclocal.m4.
+* aclocal-1.9: (automake-1.9)Invoking aclocal.          Generating aclocal.m4.
 @end direntry
 
 @titlepage
@@ -8789,7 +8789,7 @@ Automake.
 @table @asis
 @item 1994-09-19 First CVS commit.
 
-If we can trust the CVS repository, David J.@tie{}MacKenzie (djm) started
+If we can trust the CVS repository, David J. MacKenzie (djm) started
 working on Automake (or AutoMake, as it was spelt then) this Monday.
 
 The first version of the @command{automake} script looks as follows.
@@ -9090,7 +9090,7 @@ lasts.
 
 @item 1996-10-10 Kevin Dalley packages Automake 1.0 for Debian GNU/Linux.
 
-@item 1996-11-26 David J.@tie{}MacKenzie releases Autoconf 2.12.
+@item 1996-11-26 David J. MacKenzie releases Autoconf 2.12.
 
 Between June and October, the Autoconf development is almost staled.
 Roland McGrath has been working at the beginning of the year.  David
@@ -9310,7 +9310,7 @@ Autoconf 2.13 compatibility yet, this ex
 used when an environment variable was set, the traditional
 grep-scanner being still the default.
 
-@item 2001-04-25 Gary V.@tie{}Vaughan releases Libtool 1.4
+@item 2001-04-25 Gary V. Vaughan releases Libtool 1.4
 
 It has been more than two years since Automake 1.4, CVS Automake has
 suffered lot's of heavy changes and still is not ready for release.
@@ -9319,7 +9319,7 @@ Libtool 1.4 had to be distributed with a
 @item 2001-05-08 Automake 1.4-p1
 @itemx 2001-05-24 Automake 1.4-p2
 
-Gary V.@tie{}Vaughan, the principal Libtool maintainer, makes a ``patch
+Gary V. Vaughan, the principal Libtool maintainer, makes a ``patch
 release'' of Automake:
 
 @quotation
@@ -9822,7 +9822,7 @@ The number of test cases in the test sui
 @end table
 
 @multitable {8888-88-88} {8.8-p8} {8888} {888} {8888} {8888 (88)} {8888 (88)} {888} {888}
-@headitem Date   @tab Rel @tab am @tab acl @tab pm @tab @file{*.am} @tab m4 @tab doc @tab t
+@item Date   @tab Rel @tab am @tab acl @tab pm @tab @file{*.am} @tab m4 @tab doc @tab t
 @item 1994-09-19 @tab CVS    @tab  141 @tab     @tab      @tab  299 (24) @tab           @tab     @tab
 @item 1994-11-05 @tab CVS    @tab  208 @tab     @tab      @tab  332 (28) @tab           @tab     @tab
 @item 1995-11-23 @tab 0.20   @tab  533 @tab     @tab      @tab  458 (35) @tab           @tab   9 @tab
